### PR TITLE
HCCDOC-2054 Insights API cheatsheet tile

### PR DESCRIPTION
**Pull request by CodeDuchess**
**Insights API cheatsheet**

- Adding a tile to increase customer awareness of and accessibility to the cheatsheet
- This is a tile that provides **an external link** to the **cheatsheet** on **developers.redhat.com.** 
- Title: **Insights API cheatsheet**
- Description: Reference the Insights API cheatsheet to learn about authentication, work with Insights applications, execute remediations, receive notifications and intergrate with third party applications.
- Link: []((https://developers.redhat.com/cheat-sheets/red-hat-insights-api-cheat-sheet))

### CODE REVIEW

**Comment by CodeDuchess** [comment]
@Hyperkid123 Are you still the reviewer for this work? Please advise if I should be tagging other team members here. Thanks.

**Comment by dayleparker** [comment]
Hey @CodeDuchess !
I saw this mentioned on Slack and took a quick look, noticed a typo to be fixed: intergrate

Another thing to be mindful about is the character count of the description - while the [Best Practices doc](https://www.uxd-hub.com/entries/resource/best-practices-for-writing-quick-starts) advises a 115 character limit, we've found the text can get truncated even with 115 characters (Nacia is aware - did she review?). It would be good to shorten the description before merging as it's longer than that (it helps to paste the yaml into the React preview tool to see how the card looks). Let me know if you need a hand!

File: docs/quickstarts/insights-api-cheatsheet/insights-api-cheatsheet.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  name: insights-api-cheatsheet
+  externalDocumentation: true
+spec:
+  version: 0.1
+  type:
+    text: Documentation
+    color: orange
+  displayName: Insights API Cheatsheet
+  icon: ~
+  description: Reference the Insights API cheatsheet to work with Insights applications.
```
Comment (CodeDuchess) [inline docs/quickstarts/insights-api-cheatsheet/insights-api-cheatsheet.yml:13]:
```suggestion
  description: Obtain system details and interact with individual Insights applications.
```

**Comment by dayleparker** [comment]
Looks great to me, Dana! Thanks for taking the time to review & edit the description.

File: docs/quickstarts/insights-api-cheatsheet/insights-api-cheatsheet.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  name: insights-api-cheatsheet
+  externalDocumentation: true
+spec:
+  version: 0.1
+  type:
+    text: Documentation
+    color: orange
+  displayName: Insights API Cheatsheet
+  icon: ~
+  description: Obtain system details and interact with individual Insights applications.
```
Comment (CodeDuchess) [inline docs/quickstarts/insights-api-cheatsheet/insights-api-cheatsheet.yml:13]:
```suggestion
  description: Obtain system details and interact with individual Insights services.
```

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**